src/frontend/js/index.tsx

```diff
@@ -25,7 +25,9 @@ async function render() {
   // Only move on with anything if there are actually components to render
   if (richieReactSpots.length) {
     // Determine the BCP47/RFC5646 locale to use
-    const locale = document.querySelector('html')!.getAttribute('lang');
+    // This standard specifies that some locale implementations may use '_' instead of '-'
```

Comment:
Where did you see that ? I don't see usage of `_` in the RFC5646

Reply:
Whoops you are right. I search a little bit quickly, this concerns OpenID that can use locale with BCP47/RFC4646 format and some implementation may use an underscore instead of a dash.